libavcodec/qsvenc_h264.c

```diff
@@ -124,7 +124,9 @@ static const AVOption options[] = {
     { "2x"                     , NULL, 0, AV_OPT_TYPE_CONST, { .i64 = MFX_LOOKAHEAD_DS_2x      }, INT_MIN, INT_MAX,     VE, "look_ahead_downsampling" },
     { "4x"                     , NULL, 0, AV_OPT_TYPE_CONST, { .i64 = MFX_LOOKAHEAD_DS_4x      }, INT_MIN, INT_MAX,     VE, "look_ahead_downsampling" },
 #endif
-
```

Comment:
Try not to introduce irrelevant changes for empty lines.
And why not add the new option at the end?

Reply:
Thanks for review. Fixed.

Reply:
This feature is supported for AVC and MPEG2 in MSDK, but only AVC has been tested.